app/elements/common/settings/settings-page.html

```diff
@@ -65,7 +72,8 @@
 					<div>
 						<div class="list-item">
 							<div class="item-label">Email</div>
-							<paper-input value="{{user.email}}" no-label-float></paper-input>
+							<paper-input value="{{user.email}}" no-label-float type="email"
+							required auto-validate error-message="Please enter a valid email 'example@email.com'"></paper-input>
```

Comment:
For examples and docs please use domain names compliant with [RFC 2606](https://tools.ietf.org/html/rfc2606#section-3). "email.com" is a valid domain owned by somebody else. Use "me@example.com" or "me@mail.example" or something similar.